### DISCUSSION TITLE
vanilla javascript documentation

### DISCUSSION THREAD

**Discussion by jdgamble555** [urql-graphql/urql#1355] [Q&A] [answered]
I don't see any documentation for providing the basic client in **plain javascript** or **typescript**:

https://formidable.com/open-source/urql/docs/basics/getting-started/

```typescript
import { createClient } from 'urql';

const client = createClient({
  url: MY_URL,
});
```

Now how do I provide the client?  I see **Vue**, **Svelte**, and **React** documentation?  Regular JS or TS?

Thanks,
J

**Comment by kitten** [comment, marked as answer]
That's mostly addressed here: https://formidable.com/open-source/urql/docs/concepts/core-package/
The concepts are similar to framework use and the API docs support this.

But basically it comes down to using the `Client` directly

**Reply by jdgamble555** [reply]
Got it, thank you!  I was trying to import ```urql``` instead of ```@urql/core```!
